### DISCUSSION TITLE
API to reload script from disk

### DISCUSSION THREAD

**Discussion by loop333** [jcorporation/myMPD#1326] [Ideas]
If you want to use external editor to develop scripts you have to have an option to invalidate lua byte cache or reload script from disk. Now we have only MYMPD_API_SCRIPT_SAVE and params.content to update script in mympd cache before its execution

**Comment by loop333** [comment]
or maybe checking file timestamp before execution to reload new script from disk

**Comment by jcorporation** [comment]
An API to reload scripts is a good idea. It will do the same thing as on myMPD startup. It will discover new scripts, reloads existing ones and remove deleted scripts.

**Comment by jcorporation** [comment]
Do you need a button in the gui or is an API method good enough?

**Comment by loop333** [comment]
I prefer to use external toolchain ;) to work with scripts ) So I'm going to use API
